docs/config.md

```diff
@@ -577,30 +730,66 @@ It will run the model `vgg_19_graphdef` over combinations of batch sizes `[1,2,3
 
-### `<triton-server-flags>`
+## The `analysis_models` fields and `<analysis-model>`
+The `--analysis-models` argument can be provided as a list of strings (names of models) from the CLI interface, or as a more complex `<analysis-model>` object but only through the YAML configuration file. The model object can contain `<objectives>` and `<constraints>`. An example looks like:
 
-This section of the config allows fine-grained control over the flags passed to the Triton instances launched by Model Analyzer when it is running in the `docker` or `local` Triton launch modes. Any argument to the server can be specified here.
+```yaml
+analysis_models:
+  model_1:  
+    constraints:
+        gpu_used_memory:
+            max: 200
+    objectives:
+    - perf_throughput
+```
 
-#### Example
+Multiple models can be specified under the `analysis_models` key as well.
 
 ```yaml
-model_repository: /path/to/model/repository/
-model_names:
-  model_1:
-    parameters:
-        batch_sizes:
-            start: 4
-            stop: 9
-        concurrency:
-            - 2
-            - 4
-            - 8
-triton_server_flags:
-    strict-model-config: False
-    log-verbose: True
+analysis_models:
+  model_1:  
+    constraints:
+        gpu_used_memory:
+            max: 200
+  model_1:  
+    constraints:
+        perf_latency:
+            max: 80
+objectives:
+- perf_throughput
 ```
 
-**Important Notes**: 
-1. The section name contains underscores `triton_server_flags` as seen in the example above. However the arguments themselves should contain hyphens as seen with `strict-model-config`.
-2. The Model Analyzer also provides certain arguments to the `tritonserver` instances it launches. These ***cannot*** be overriden by providing those arguments in this section. An example of this is `http-port`, which is an argument to Model Analyzer itself.
+## The `report_model_configs` fields and `<report-model-config>`
+The `--report-model-configs` argument can be provided as a list of strings (names of models) from the CLI interface, or as a more complex `<report-model-config>` object but only through the YAML configuration file. The model object can contain `<plots>` objects. An example looks like:
+
+```yaml
+report_model_configs:
+  model_1_i0:
+    throughput_v_latency:
+      title: Title
+      x_axis: perf_latency
+      y_axis: perf_throughput
+      monotonic: True
+```
+
+Multiple models can be specified under the `analysis_models` key as well.
+
+```yaml
+report_model_configs:
+  model_1_i0:
+    throughput_v_latency:
+        title: Title
+        x_axis: perf_latency
+        y_axis: perf_throughput
+        monotonic: True
+  model_2_i0:
+    gpu_mem_v_latency:
+        title: Title
+        x_axis: perf_latency
+        y_axis: gpu_used_memory
+        monotonic: False
+```
+
+## Config Defaults
 
+Up to date config defaults can be found in [config_defaults.py](../model_analyzer/config/input/config_defaults.py)
```

Comment:
The config defaults are included in the list of parameters described above. Is there any reason why we point them to this file?

Reply:
Several config defaults are too large to include here.